addons/l10n_in/models/template_in.py

```diff
@@ -35,10 +35,21 @@ def _get_in_res_company(self):
                 'expense_account_id': 'p2107',
                 'income_account_id': 'p20011',
                 'l10n_in_withholding_account_id': 'p100595',
+                'account_stock_journal_id': 'inventory_valuation',
+                'account_stock_valuation_id': 'p10031',
                 'tax_calculation_rounding_method': 'round_per_line',
             },
         }
 
+    @template('in', 'account.account')
+    def _get_be_account_account(self):
```

Comment:
```suggestion
    def _get_in_account_account(self):
```
We might override the `l10n_be` method in `l10n_in` or vice-versa, whichever module is installed or updated later